coldfront/plugins/qumulo/management/commands/check_billing_cycles.py

```diff
@@ -61,15 +62,18 @@ def calculate_prepaid_expiration(
         name="prepaid_expiration"
     )
     logger.warn(f"Calculation prepaid expiration")
+    
     if bill_cycle == "prepaid" and prepaid_expiration == None:
         prepaid_billing_start = datetime.strptime(prepaid_billing_start, "%Y-%m-%d")
+        last_day_of_month = False
         prepaid_months = int(prepaid_months)
-        prepaid_until = datetime(
-            prepaid_billing_start.year
-            + (prepaid_billing_start.month + prepaid_months - 1) // 12,
-            (prepaid_billing_start.month + prepaid_months - 1) % 12 + 1,
-            prepaid_billing_start.day,
-        )
+        prepaid_until = prepaid_billing_start + relativedelta(months=prepaid_months)
+        last_day_of_month = (prepaid_billing_start.day == calendar.monthrange(prepaid_billing_start.year, prepaid_billing_start.month)[1])
```

Comment:
```suggestion
        is_last_day_of_month = (prepaid_billing_start.day == calendar.monthrange(prepaid_billing_start.year, prepaid_billing_start.month)[1])
```

Reason: Makes it clear variable is a boolean that answers a questions and not a number that stores information